kube-controllers/pkg/controllers/node/ipam.go

```diff
@@ -250,18 +293,58 @@ func (c *ipamController) acceptScheduleRequests(stopCh <-chan struct{}) {
 	for {
 		// Wait until something wakes us up, or we are stopped.
 		select {
+		case <-c.nodeDeletionChan:
```

Comment:
Might be good to service both channels during the consolidation window?  (But this is likely a lot better than it was!)